vyper/parser/function_definitions/__init__.py

```diff
@@ -0,0 +1,92 @@
+from vyper.exceptions import (
+    FunctionDeclarationException,
+)
+from vyper.parser.context import (
+    Constancy,
+    Context,
+)
+from vyper.parser.function_definitions.parse_private_function import (
+    parse_private_function,
+)
+from vyper.parser.function_definitions.parse_public_function import (
+    parse_public_function,
+)
+from vyper.parser.memory_allocator import (
+    MemoryAllocator,
+)
+from vyper.signatures import (
+    FunctionSignature,
+)
+from vyper.utils import (
+    calc_mem_gas,
+)
+
+
+# Is a function the initializer?
+def is_initializer(code):
```

Comment:
not sure it's best practice for `__init__.py` to actually define stuff, I thought they should just import things? So maybe this can go into `vyper/parser/function_definitions/utils.py`.